README.md

```diff
@@ -7,916 +7,53 @@
-
-### Updating and restarting services
-
-When you update service configuration (by adding a layer), the services changed won't be automatically restarted. To restart them and bring the service state in sync with the new configuration, use `pebble replan`.
-
-The "replan" operation restarts `startup: enabled` services whose configuration have changed between when they started and now; if the configuration hasn't changed, replan does nothing. Replan also starts `startup: enabled` services that have not yet been started.
-
-Here is an example, where `srv1` is a service that has `startup: enabled`, and `srv2` does not:
-
-```
-$ pebble replan
-2023-04-25T15:06:50+02:00 INFO Service "srv1" already started.
-$ pebble add lay1 layer.yaml  # update srv1 config
-Layer "lay1" added successfully from "layer.yaml"
-$ pebble replan
-Stop service "srv1"
-Start service "srv1"
-$ pebble add lay2 layer.yaml  # change srv2 to "startup: enabled"
-Layer "lay2" added successfully from "layer.yaml"
-$ pebble replan
-2023-04-25T15:11:22+02:00 INFO Service "srv1" already started.
-Start service "srv2"
 ```
 
-If you want to force a service to restart even if its service configuration hasn't changed, use `pebble restart <service>`.
-
-### Service dependencies
-
-Pebble takes service dependencies into account when starting and stopping services. When Pebble starts a service, it also starts the services which that service depends on (configured with `requires`). Conversely, when stopping a service, Pebble also stops services which depend on that service.
-
-For example, if service `nginx` requires `logger`, `pebble start nginx` will start both `nginx` and `logger` (in an undefined order). Running `pebble stop logger` will stop both `nginx` and `logger`; however, running `pebble stop nginx` will only stop `nginx` (`nginx` depends on `logger`, not the other way around).
-
-When multiple services need to be started together, they're started in order according to the `before` and `after` configuration, waiting 1 second for each to ensure the command doesn't exit too quickly. The `before` option is a list of services that this service must start before (it may or may not "require" them). Or if it's easier to specify this ordering the other way around, `after` is a list of services that this service must start after.
+Read more about Pebble's general model [here](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/general-model/).
 
-Note that currently, `before` and `after` are of limited usefulness, because Pebble only waits 1 second before moving on to start the next service, with no additional checks that the previous service is operating correctly.
+For a slightly longer introduction, you can also go through the [official tutorial](https://canonical-pebble.readthedocs-hosted.com/en/latest/tutorial/getting-started/)
```

Comment:
```suggestion
For a hands-on introduction to Pebble, we recommend going through the [tutorial](https://canonical-pebble.readthedocs-hosted.com/en/latest/tutorial/getting-started/).
```

Reply:
Because "longer" doesn't necessarily mean "better" so hands-on might be more appealing since users know they'll be getting their feet wet.

Thanks.